esp/templates/elements/html

```diff
@@ -46,6 +46,7 @@
 
     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/react/0.13.3/react.js"></script>
     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/react/0.13.3/JSXTransformer.js"></script>
+    <script type="text/javascript" src="https://rawgit.com/arqex/react-json-table/master/build/react-json-table.min.js"></script>
```

Comment:
Since this is only used on one page, I'd just put it there.  Also, I would include a specific commit rather than master, just in case there is at some point a breaking change.